### PR TITLE
TCM-112 | Build dummy video screen

### PR DESCRIPTION
**Pull request by halilyesilyurt**
### Summary

- [x] Completed dummy video screen.
- [x] Added video screen navigation to test utility.

### Issue

[TCM-112](https://hyfnla.atlassian.net/browse/TCM-112)

### Video/Screenshot

| TAB_WORKOUT| TAB_COLLECTIONS|
|--------|--------|
|video_1|video_2|

DEMO

https://user-images.githubusercontent.com/30659854/118462475-627dd580-b707-11eb-8e8b-9a43ede3e9f6.mp4

### CODE REVIEW

File: app/src/main/res/layout/fragment_video.xml [hunk 1]
```diff
@@ -1,14 +1,116 @@
...
+            android:textColor="@color/black"
+            android:textSize="16sp"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent" />
+
+        <com.trx.consumer.common.CommonButton
+            android:id="@+id/btnCollections"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:fontFamily="@font/atcarquette_semibold"
```
Comment (onodude) [inline app/src/main/res/layout/fragment_video.xml:36]: font should be bold

File: app/src/main/res/layout/fragment_video.xml [hunk 2]
```diff
@@ -1,14 +1,116 @@
...
+            android:text="@string/video_tab_collections_label"
+            android:textAllCaps="true"
+            android:textColor="@color/greyLight"
+            android:textSize="16sp"
```
Comment (onodude) [inline app/src/main/res/layout/fragment_video.xml:40]: text size is 13sp

File: app/src/main/res/layout/row_video_detail.xml [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+        tools:text="Foll Body Power Pump" />
+
+    <com.trx.consumer.common.CommonLabel
+        android:id="@+id/lblSubtitle"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="15dp"
+        android:layout_marginEnd="15dp"
+        android:layout_marginBottom="15dp"
+        android:fontFamily="@font/atcarquette_bold"
```
Comment (onodude) [inline app/src/main/res/layout/row_video_detail.xml:57]: should be semi bold

File: app/src/main/res/layout/row_video_detail.xml [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+        tools:text="60 MIN" />
+
+    <com.trx.consumer.common.CommonLabel
+        android:id="@+id/lblTitle"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="15dp"
+        android:layout_marginEnd="15dp"
+        android:layout_marginBottom="5dp"
+        android:fontFamily="@font/atcarquette_bold"
```
Comment (onodude) [inline app/src/main/res/layout/row_video_detail.xml:42]: font should be semi bold

File: app/src/main/res/layout/fragment_video.xml (hunk 1 above)
Comment (halilyesilyurt) [reply app/src/main/res/layout/fragment_video.xml]: Completed.

File: app/src/main/res/layout/row_video_detail.xml (hunk 3 above)
Comment (halilyesilyurt) [reply app/src/main/res/layout/row_video_detail.xml]: Completed.

File: app/src/main/res/layout/fragment_video.xml (hunk 2 above)
Comment (halilyesilyurt) [reply app/src/main/res/layout/fragment_video.xml]: Completed.

**Comment by halilyesilyurt** [comment]
changed pr images, and added demo.

**Review by VivekV95 — APPROVED** [review]

**Review by onodude — APPROVED** [review]

**Comment by onodude** [comment]
thanks @halilyesilyurt @VivekV95

**Pull request merged**